helper/impls/helper-impl-14/src/main/java/com/linkedin/avroutil1/compatibility/avro14/codec/CompatibleJsonEncoder.java

```diff
@@ -48,8 +48,8 @@
    */
   protected BitSet isEmpty = new BitSet();
 
-  public CompatibleJsonEncoder(Schema sc, OutputStream out, boolean useFqcns) throws IOException {
-    this(sc, getJsonGenerator(out), useFqcns);
+  public CompatibleJsonEncoder(Schema sc, OutputStream out, boolean pretty, boolean useFqcns) throws IOException {
```

Comment:
for backwards compatibility please create an overload of this method with the old signature as well